zerver/lib/user_topics.py

```diff
@@ -117,8 +118,10 @@ def set_user_topic_visibility_policy_in_database(
     visibility_policy: int,
     recipient_id: Optional[int] = None,
     last_updated: Optional[datetime.datetime] = None,
-    ignore_duplicate: bool = False,
-) -> None:
+) -> bool:
+    # returns True if it modifies the database and False otherwise.
+    database_changed: bool = True
```

Comment:
Don't define this variable -- it's never usefully mutated. We should just `return True` / `return False` where needed.